cli/cmd/install-cni-plugin.go

```diff
@@ -120,6 +123,11 @@ assumes that the 'linkerd install' command will be executed with the
 	cmd.PersistentFlags().StringVar(&options.logLevel, "cni-log-level", options.logLevel, "Log level for the cni-plugin.")
 	cmd.PersistentFlags().StringVar(&options.destCNINetDir, "dest-cni-net-dir", options.destCNINetDir, "Directory on the host where the CNI configuration will be placed.")
 	cmd.PersistentFlags().StringVar(&options.destCNIBinDir, "dest-cni-bin-dir", options.destCNIBinDir, "Directory on the host where the CNI plugin binaries reside.")
+	cmd.PersistentFlags().BoolVar(
+		&options.useWaitFlag,
+		"use-wait-flag",
+		options.useWaitFlag,
+		"Configures the CNI plugin to use the \"-w\" flag for the iptables command.")
```

Comment:
Maybe put in the description the default value (false) in parenthesis?

Reply:
Added default value to help in [this commit](https://github.com/linkerd/linkerd2/commit/43cb598ca5a23b736a02cf3a68521a6ef9228222)